src/all/pornpics/src/eu/kanade/tachiyomi/extension/all/pornpics/PornPicsExt.kt

```diff
@@ -0,0 +1,49 @@
+package eu.kanade.tachiyomi.extension.all.pornpics
+
+import okhttp3.HttpUrl
+
+internal const val QUERY_PAGE_SIZE = 19
+
+internal fun HttpUrl.Builder.addQueryParameter(encodedName: String, encodedValue: Int) =
+    addQueryParameter(encodedName, encodedValue.toString())
+
+internal fun HttpUrl.Builder.addQueryPageParameter(page: Int) =
```

Comment:
So it lines up with `.addQueryParameterPage()`:
```suggestion
internal fun HttpUrl.Builder.addQueryParameterPage(page: Int) =
```